### ISSUE TITLE
Does not work with private packages

### ISSUE THREAD

**Issue by stefaneg** [jesseditson/npm-package#3]
Npm registry wants a bearer token to give you information about private packages. The following code works for me:

```javascript
const rc = require('rc');
const registryUrl = require('registry-url');
const request = require('superagent');
const url = require('url');

function registryPackage (name, callback) {

    const npmConf = rc('npm', {});
    let npmAuthenticationToken = npmConf['//registry.npmjs.org/:_authToken'];

    let packageInfoRequest = request.get(url.resolve(registryUrl(), name));

    if (npmAuthenticationToken) {
        packageInfoRequest.set('Authorization', `Bearer ${npmAuthenticationToken}`);
    }
    packageInfoRequest.end(function(err, response) {
        callback(err, response.body);
    });
}

module.exports = registryPackage;
```

**Comment by jesseditson** [comment]
Awesome - I know npm has changed their private packages quite a lot since this lib was last published. The above code looks good to me, would you like to PR it so you get credit? If not I'd be happy to add it and publish a new version.

**Comment by stefaneg** [comment]
Way too busy with other stuff, so please go ahead :-)

**Comment by jesseditson** [comment]
Published in [0.1.0](https://www.npmjs.com/package/npm-package-info) - thanks!
